### PR TITLE
Story Map

### PR DESCRIPTION
**Pull request by IhsenBouallegue**
This PR adds:
- Base layout of Story Map
- Drag and drop functionality

### CODE REVIEW

File: project-canvas/src/components/StoryMapView/Cards/SubActionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  children: string
+  editSubAction: ({ id, title }: SubAction) => void
+} & PaperProps) {
+  return (
+    <ItemCard
+      id={id}
+      index={index}
+      editItem={() => editSubAction({ id, title: "NEW SUB BABY" })}
+      itemType="subAction"
+      m="10px"
```
Comment (bic-marz) [inline project-canvas/src/components/StoryMapView/Cards/SubActionCard.tsx:23]: don't forget this dude

File: project-canvas/src/components/StoryMapView/Cards/ActionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  children: string
+  editAction: ({ id, title }: Action) => void
+} & PaperProps) {
+  return (
+    <ItemCard
+      id={id}
+      index={index}
+      editItem={() => editAction({ id, title: "NEWWW" } as Action)}
+      itemType="action"
+      m="10px"
```
Comment (bic-marz) [inline project-canvas/src/components/StoryMapView/Cards/ActionCard.tsx:23]: and this

File: project-canvas/src/components/StoryMapView/CaseSubActions.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                {provided.placeholder}
+              </Stack>
+            )}
+          </StrictModeDroppable>
+        ))}
+        <StrictModeDroppable key="add" droppableId="add" isDropDisabled>
+          {(provided) => (
+            <Stack
+              w="162px"
+              spacing={0}
```
Comment (bic-marz) [inline project-canvas/src/components/StoryMapView/CaseSubActions.tsx:57]: flagged for you

**Review by bic-marz — APPROVED** [review]

File: project-canvas/src/components/StoryMapView/Cards/ActionCard.tsx (hunk 2 above)
Comment (IhsenBouallegue) [reply project-canvas/src/components/StoryMapView/Cards/ActionCard.tsx:23]: done

**Pull request merged**